lib/rack/tracker/criteo/criteo.rb

```diff
@@ -0,0 +1,31 @@
+class Criteo <  Rack::Tracker::Handler
+  class Event < OpenStruct
+    def write
+      to_h.to_json
+    end
+  end
+
+  self.position = :body
+
+  def tracker_options
```

Comment:
since criteo supports setting those as events, they can be then triggered as events and we don't have to jump through hoops like we did for GA in #15